### ISSUE TITLE
Research if using Roman numerals with World War II is an accessibility issue and if there is a fix for it

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#21959]
## Issue Description
We use Roman numerals on our site when referring to World War II. These numbers are most likely read by screen readers as "ii." While it's AP Style to write it this way, if this is an accessibility issue, we should reevaluate how we write it on our site. Or determine if we could use aria labels or some other coding to allow it to read as the number 2. 

## Tasks
- [x] Reach out to accessibility team to determine if WWII is a screen reader issue
- [ ] If an issue, work with accessibility team to discuss a fix
- [ ] Update style guide if need be

## Acceptance Criteria
- [ ] World War II written on our site is no longer an accessibility issue

**Comment by RLHecht** [comment]
@1Copenut Hoping you or someone on your team could help us with this! Our style, which follows AP style is to use Roman numerals for World War II. Is there a way we could continue to do this with aria labels or some other coding that I don't know that would allow the screen reader to pronounce it as 2?

**Comment by 1Copenut** [comment]
@RLHecht This is a tough one because some screen readers handle them correct-ish (NVDA) and others like JAWS don't do so well. VoiceOver is somewhere in the middle.

I mocked up this Codepen https://codepen.io/tpierce_402/pen/RwKWrOm to start testing different possibilities, but haven't put a lot of time into testing. 

/cc @joshkimux for awareness.

**Comment by joshkimux** [comment]
Would love to hear @AngelaFowler82 's thoughts on roman numerals too!

**Comment by AngelaFowler82** [comment]
In order to test this, I typed WWII in a word document. NVDA correctly pronounced it World War 2, while JAWS said w w I i. 

I am in contact with Freedom Scientific to determine why JAWS does not pronounce this correctly. It may be a setting I need to change.

VoiceOver on the iPhone pronounces it correctly.

**Comment by RLHecht** [comment]
So interesting! Thank you all for looking into this. FYSA  @bethpottsVADEPO

**Comment by AngelaFowler82** [comment]
Hello all,

At this point, the only way JAWS will pronounce WWII correctly is if the user changes the pronunciation using the dictionary manager. I think this should be changed within JAWS, as all other screen readers I’ve tested pronounce it correctly and I can think of no other circumstance in which WWII is used. I have made this suggestion to the manufacturers of JAWS.

Angela

**Comment by 1Copenut** [comment]
@AngelaFowler82 Thank you for this explanation and reaching out to Freedom Scientific about JAWS. Seems the current handling of Roman numerals is as good as it can be without vendor improvement for JAWS.

**Comment by RLHecht** [comment]
Thank you all for working on this so quickly! Looks like we can close this ticket out since no further action is needed in regards to updating our content.
